cli/cmd/cloud_account.go

```diff
@@ -20,52 +25,47 @@ var (
 
 	// cloudAccountsListCmd represents the list sub-command inside the cloud accounts command
 	cloudAccountListCmd = &cobra.Command{
-		Use:   "list",
-		Short: "List all available cloud account integrations",
-		Args:  cobra.NoArgs,
-		RunE: func(_ *cobra.Command, _ []string) error {
-			var (
-				cloudAccounts api.CloudAccountsResponse
-				err           error
-			)
-
-			if cloudAccountType != "" {
-				caType, found := api.FindCloudAccountType(cloudAccountType)
-				if !found {
-					return errors.Errorf("unknown cloud account type '%s'", cloudAccountType)
-				}
-				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.ListByType(caType)
-			} else {
-				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.List()
-			}
-			if err != nil {
-				return errors.Wrap(err, "unable to get cloud accounts")
-			}
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all available cloud account integrations",
+		Args:    cobra.NoArgs,
+		RunE:    cloudAccountList,
+	}
 
-			if len(cloudAccounts.Data) == 0 {
-				cli.OutputHuman("No cloud accounts found.\n")
-				return nil
-			}
+	// cloudAccountShowCmd represents the show sub-command inside the cloud accounts command
+	cloudAccountShowCmd = &cobra.Command{
+		Use:     "show",
+		Aliases: []string{"get"},
+		Short:   "Show a single cloud account integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    cloudAccountShow,
```

Comment:
This command is NOT showing the `state` field that helps customers troubleshoot cloud account integrations. 

The command `lacework int show GUID` is currently showing this information:
<img width="786" alt="Screen Shot 2022-09-07 at 12 32 06 AM" src="https://user-images.githubusercontent.com/5712253/188816908-714a2176-c2af-4a7c-8e70-ef746e270ed7.png">

Reply:
Could you send me the id of the cloud account you are testing with. In this refactor I took the opportunity to remove the json output, and move everything into the table format. See attached screenshots

New Output:
![image](https://user-images.githubusercontent.com/75614232/188850614-ab3d11b3-f79c-477b-b674-0923eb17954d.png)

Old Output:
![image](https://user-images.githubusercontent.com/75614232/188850681-1331bf6e-035f-48b3-b728-964b72ca4ea2.png)